src/main/scala/com/microsoft/ml/spark/isolationforest/IsolationForest.scala

```diff
@@ -0,0 +1,65 @@
+// Copyright (C) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in project root for information.
+
+package com.microsoft.ml.spark.isolationforest
+
+import com.microsoft.ml.spark.core.env.InternalWrapper
+import org.apache.spark.ml.param.ParamMap
+import org.apache.spark.ml.util.{DefaultParamsReadable, DefaultParamsWritable, Identifiable, MLReadable, MLReader, MLWritable, MLWriter}
+import org.apache.spark.ml.{ComplexParamsReadable, ComplexParamsWritable, Estimator, Model}
+import com.linkedin.relevance.isolationforest.{IsolationForestModelReadWrite, IsolationForestParams,
```

Comment:
is this source available anywhere?  just seems a bit weird to have an estimator with almost no implementation in mmlspark

Reply:
Source is available, but LinkedIn would like to continue maintaining the repo separately. This wrapper is mainly there so that the pyspark/R wrappers are generated.

If there's an alternative route, let me know.

Reply:
oh ok, thanks for explaining, that makes sense!